docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -143,6 +143,11 @@ In this example, a dedicated pool of 32MB is allocated for clustered-cache from
 <7> Configures another cache (`shared-cache-2`) that shares the resource pool (`resource-pool-a`) with `shared-cache-1`.
 <8> Creates fully initialized cache manager with the clustered caches.
 
+NOTE:  When multiple caches are sharing a shared pool, eviction operation called by a cache evicts elements of that cache
```

Comment:
Am I right to say that the real thing is that a cache will reserve a block of memory from the shared pool and keep it forever? So eviction or not, this allocated memory will stay with a given cache and that it's. If I am right, that's probably what you should explain.

Reply:
revised it, please let me know if it sounds better now